main.py

```diff
@@ -208,6 +215,7 @@ def campaign_group_page(campaign_group=None, pwg=None):
     if only_with_miniaod:
         sql_query += ' AND miniaod != ""'
 
+    sql_query += ' LIMIT %s OFFSET %s' % (page_size, (page-1)*page_size)
```

Comment:
Hm, I think you should also add `ORDER BY dataset`. Because now rows in table are almost randomly sorted, e.g. Zprime is somewhere between Bulk and Tprime.